src/monitor.py

```diff
@@ -0,0 +1,75 @@
+from screeninfo import get_monitors
+
+
+class Monitor:
+    def __init__(self, name, width, height, x, y, primary, wallpaper=None):
+        self.width = width
+        self.height = height
+        self.wallpaper = wallpaper
+        self.primary = primary
+        self.x = x
+        self.y = y
+        self.name = name
+
+    def set_wallpaper(self, wallpaper):
+        self.wallpaper = wallpaper
+
+    def __str__(self):
+        return f"Monitor(name={self.name}, x={self.x}, y={self.y}, width={self.width}, height={self.height}, primary={self.primary}, wallpaper={self.wallpaper})"
+
+
+class MonitorInfo:
+    @staticmethod
+    def get_unique_monitor_count():
+        monitors = get_monitors()
+        unique_names = set()
```

Comment:
Just out of curiosity, can the monitors have the same name? If there isn't, maybe we don't need to make a set.

Reply:
With gdk, we dont have to use it i think, it can return monitor count with 
```python
def get_unique_monitor_count():
        display = Gdk.Display.get_default()
        return display.get_n_monitors()
```